lib/services/debug-service.ts

```diff
@@ -35,9 +37,16 @@ class DebugService extends EventEmitter {
 		// After we find a way to check on iOS we should use it here.
 		const isAppRunning = true;
 		let result: string[];
-		debugOptions.chrome = !debugOptions.client;
+		debugOptions.chrome = true;
```

Comment:
There's no plans to support AppInspector at the moment, I want to be sure that passing different options will always use Chrome DevTools. Once we decide to change this, we'll modify the code and the tests.